### ISSUE TITLE
Calendar multiget should remove empty params

### ISSUE THREAD

**Issue by maksimlya** [natelindev/tsdav#224]
```
export const calendarMultiGet = async (params: {
  url: string;
  props: ElementCompact;
  objectUrls?: string[];
  timezone?: string;
  depth: DAVDepth;
  filters?: ElementCompact;
  headers?: Record<string, string>;
  headersToExclude?: string[];
  fetchOptions?: RequestInit;
}): Promise<DAVResponse[]> => {
  const { url, props, objectUrls, filters, timezone, depth, headers, headersToExclude, fetchOptions = {} } = params;
  return collectionQuery({
    url,
    body: {
      'calendar-multiget': {
        _attributes: getDAVAttribute([DAVNamespace.DAV, DAVNamespace.CALDAV]),
        [`${DAVNamespaceShort.DAV}:prop`]: props,
        [`${DAVNamespaceShort.DAV}:href`]: objectUrls,
        ...conditionalParam('filter', filters),
        timezone,
      },
    },
    defaultNamespace: DAVNamespaceShort.CALDAV,
    depth,
    headers: excludeHeaders(headers, headersToExclude),
    fetchOptions,
  });
};
```

The following function should be refactored to 

```
export const calendarMultiGet = async (params: {
  url: string;
  props: ElementCompact;
  objectUrls?: string[];
  timezone?: string;
  depth: DAVDepth;
  filters?: ElementCompact;
  headers?: Record<string, string>;
  headersToExclude?: string[];
  fetchOptions?: RequestInit;
}): Promise<DAVResponse[]> => {
  const { url, props, objectUrls, filters, timezone, depth, headers, headersToExclude, fetchOptions = {} } = params;
  return collectionQuery({
    url,
    body: {
      'calendar-multiget': cleanupFalsy({
        _attributes: getDAVAttribute([DAVNamespace.DAV, DAVNamespace.CALDAV]),
        [`${DAVNamespaceShort.DAV}:prop`]: props,
        [`${DAVNamespaceShort.DAV}:href`]: objectUrls,
        ...conditionalParam('filter', filters),
        timezone,
      }),
    },
    defaultNamespace: DAVNamespaceShort.CALDAV,
    depth,
    headers: excludeHeaders(headers, headersToExclude),
    fetchOptions,
  });
};
```

Wrapping the body part in 'cleanupFalsy' filter. This because if for example 'timezone' is sent empty, it still creates an xml that contains timezone prop, but some providers, in my case yahoo calendar just throws 400 Bad Request:

**Bad Request:**
```
<c:calendar-multiget xmlns:d="DAV:" xmlns:c="urn:ietf:params:xml:ns:caldav">
  <d:prop>
    <d:getetag/>
    <c:calendar-data/>
  </d:prop>
  <d:href>https://caldav.calendar.yahoo.com/dav/5nl27535b2smij7yq6ilwzhycxq7vupsutlbqgr5/Calendar/131/04494773087984918.ics</d:href>
  <c:timezone/>
</c:calendar-multiget>
```

**Working one:**
```
<c:calendar-multiget xmlns:d="DAV:" xmlns:c="urn:ietf:params:xml:ns:caldav">
  <d:prop>
    <d:getetag/>
    <c:calendar-data/>
  </d:prop>
  <d:href>https://caldav.calendar.yahoo.com/dav/5nl27535b2smij7yq6ilwzhycxq7vupsutlbqgr5/Calendar/131/04494773087984918.ics</d:href>
</c:calendar-multiget>
```

**Comment by maksimlya** [comment]
Basically just copy-paste from same as you do in 'calendarQuery' function already. Thanks!

**Comment by natelindev** [comment]
make sense,i'll fix it

**Comment by natelindev** [comment]
fixed in v2.1.3
